### DISCUSSION TITLE
This if statement smells weird. Can anyone explain?

### DISCUSSION THREAD

**Discussion by krushia** [FreeCAD/FreeCAD#13881] [Code contribution discussions]
https://github.com/FreeCAD/FreeCAD/blob/c1bc3b6e9a404ce68e4480711aba1972a12241f3/src/App/Document.cpp#L1075-L1078

**Comment by chennes** [comment]
Probably a mistake introduced when the curly braces were added to the single-line conditional. Obviously only the first return is needed. Thanks for the catch -- feel free to submit a cleanup PR.

**Comment by krushia** [comment]
Thanks. Somehow confused my little brain more than it should have. I'm reviewing the whole file, will try to get a PR in when done.
